src/billing/components/PayAsYouGo/PayAsYouGo.tsx

```diff
@@ -17,35 +19,45 @@ import CancellationPanel from 'src/billing/components/PayAsYouGo/CancellationPan
 import NotificationPanel from 'src/billing/components/PayAsYouGo/NotificationPanel'
 import InvoiceLoadingWrapper from 'src/billing/components/AssetLoading/InvoiceWrapper'
 import BillingInfoWrapper from 'src/billing/components/AssetLoading/BillingInfoWrapper'
+import {PricingAlert} from 'src/billing/components/PayAsYouGo/PricingAlert'
+
+// Utils
+import {selectCurrentIdentity} from 'src/identity/selectors'
+
+const BillingPayAsYouGo: FC = () => {
+  const {account} = useSelector(selectCurrentIdentity)
+  const isdDirectSignup = account.billingProvider === 'zuora'
```

Comment:
`billingProvider` is not fetched until accessing the billing page, so you may want to add an explicit check here accounting for if the property is absent